src/components/Movies.vue

```diff
@@ -26,42 +34,67 @@ export default {
     data () {
         return {
             currentPage: null,
+            search: '',
+            go: false
         }
     },
     created () {
         this.currentPage = 1;
         this.loadPage(this.currentPage)
         .then((res)=> {
+            this.getPage(this.currentPage).forEach(element => {
+                this.loadReacts(element['id']);
+            });
+            this.go = true;
         }, (err) => {
             console.log(err.message);
         });
     },
     methods: {
-        ...mapActions('movies', ['getFirstPage', 'loadPage']),
+        ...mapActions('movies', ['getFirstPage', 'loadPage', 'loadReacts']),
         trim (text) {
-            var arr = text.trim();
-	        arr = arr.split(' ').slice(0, 20);
-            text = arr.join(' ') + '...';
-            return text;
+            return this.lodash.truncate(text, {
+                'length': 50,
+            });
         },
         nextPage(){
+            this.go = false;
             this.currentPage = this.currentPage + 1;
             this.loadPage(this.currentPage)
             .then(()=> {
+                this.getPage(this.currentPage).forEach(element => {
+                    this.loadReacts(element['id']);
+                });
+                this.go = true;
             })
             .catch((err) => {
                 console.log(err);
-            })
+            });
+        },
+        prevPage(){
+            this.currentPage = this.currentPage - 1;
         }
 
     },
     computed: {
-        ...mapGetters('movies', ['getPage', 'getAll']),
-        
+        ...mapGetters('movies', ['getPage', 'getAll', 'getReacts']),
+        filteredMovies: function () {
+            if(this.go) {
```

Comment:
if else kad imas mozes uglavnom da ga izbegnes ako obrnes uslov
if(!this.go) return this.getPage()
return filter....

Reply:
manje nestovanih ifova ce biti, i bice generalno citljivije